libr/util/graph.c

```diff
@@ -316,6 +328,23 @@ R_API RGraphNodeInfo *r_graph_create_node_info(char *title, char *body, ut64 off
 		data->title = title;
 		data->body = body;
```

Comment:
i think this is a setter and it should be responsible for owning the passed string, so better use const char* for body and title. just strdup in here and remove the frees below